src/ucp/proto/rndv.c

```diff
@@ -1086,25 +1139,34 @@ UCS_PROFILE_FUNC_VOID(ucp_rndv_receive, (worker, rreq, rndv_rts_hdr),
 
     if (UCP_DT_IS_CONTIG(rreq->recv.datatype)) {
         if ((rndv_rts_hdr->address != 0) &&
-            (ucp_rndv_is_get_zcopy(rreq->recv.mem_type, rndv_mode)) &&
```

Comment:
what if `put_zcopy` is forced by configuration?

Reply:
the address will be zero when `put_zcopy `is forced. it will follow line:1184 below